test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] Please make sure GitHub ignores this file. The reason might be because you pushed the file first before adding it to `.git ignore` file. Try deleting it. then push to GitHub. After that, create the file again and make sure it's in the` .gitignore `file before pushing it to GitHub